.env.template

```
@@ -77,15 +77,15 @@ DB_CERTIFICATE_USER=prompt-postgres
 # ============================================================================
-# Database for template server functionality
+# Database for example server functionality
 
-DB_HOST_TEMPLATE_SERVER=db-template-server
-DB_PORT_TEMPLATE_SERVER=5437
-DB_TEMPLATE_NAME=prompt
-DB_TEMPLATE_USER=prompt-postgres
-DB_TEMPLATE_PASSWORD=prompt-postgres
+DB_HOST_EXAMPLE_SERVER=db-example-server
+DB_PORT_EXAMPLE_SERVER=5437
+DB_EXAMPLE_NAME=prompt
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DB_EXAMPLE_NAME key should go before the DB_HOST_EXAMPLE_SERVER key

(UnorderedKey)